livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -202,6 +202,7 @@ async def _run(self, max_retry: int) -> None:
                         "vad_events": True,
                         "channels": self._num_channels,
                         "endpointing": self._config.endpointing,
+                        "utterance_end_ms": 1000,
```

Comment:
Should this be hardcoded?

Reply:
hrmm probably not. I think it can be Min(1000, endpointing).

> It is recommended that you set the value of utterance_end_ms to be 1000 ms or higher. This is because UtteranceEnd relies on Deepgram's interim_results feature. Deepgram's Interim Results are typically sent every one second, so using a value of less than one second for utterance_end_ms will not offer any benefits.